src/site/content/en/design-system/render-component-test.njk

```diff
@@ -0,0 +1,24 @@
+---
+permalink: '/design-system/render-component-test/index.html'
+title: 'Test'
+button: 
+  text: 'Override data'
+aside:
+  content: 'Some **markdown** in here'
```

Comment:
At some point we need to care about i18n for these. But that is not today I suspect.

Reply:
Yeh for sure. We can pass the i18n stuff in just like we do through the templates etc